crates/orchestrator/src/telemetry.rs

```diff
@@ -53,11 +53,21 @@ pub fn setup_analytics() -> Option<SdkMeterProvider> {
     }
 }
 
+fn get_otel_config() -> Option<OTELConfig> {
+    let otel_endpoint = get_env_var_optional("OTEL_COLLECTOR_ENDPOINT").expect("Failed to get OTEL_COLLECTOR_ENDPOINT");
+    let otel_service_name = get_env_var_optional("OTEL_SERVICE_NAME").expect("Failed to get OTEL_SERVICE_NAME");
+
+    match (otel_endpoint, otel_service_name) {
+        (Some(endpoint), Some(service_name)) => Some(OTELConfig { endpoint, service_name }),
+        _ => None,
```

Comment:
maybe warn that one was give and the other wasn't so skipping?

Reply:
updated to : 
```rust
 match (otel_endpoint, otel_service_name) {
        (Some(endpoint), Some(service_name)) => Some(OTELConfig { endpoint, service_name }),
        _ => {
            tracing::warn!("OTEL_COLLECTOR_ENDPOINT or OTEL_SERVICE_NAME is not set");
            None
        }
    }
 ```